CC/src/DgmOctree.cpp

```diff
@@ -2657,7 +2659,8 @@ ReferenceCloud* DgmOctree::getPointsInCellsWithSortedCellCodes(	cellCodesContain
     unsigned char bitDec2 = (areCodesTruncated ? 0 : bitDec1); //shift for the input codes
 
     cellsContainer::const_iterator p = m_thePointsAndTheirCellCodes.begin();
-    CellCode toExtractCode,currentCode = (p->theCode >> bitDec1); //pred value must be different than the first element's
+    CellCode toExtractCode;
+	CellCode currentCode = (p->theCode >> bitDec1); //pred value must be different than the first element's
```

Comment:
alignement?

Reply:
Ah - this method _getPointsInCellsWithSortedCellCodes_ uses spaces where the rest of the file uses tabs. So when I added this with a tab it's misaligned.

It's probably the same for the rest.